woocommerce/woocommerce-ios
# [Bundle products in order] Add pricedIndividually attribute to ProductBundleItem entity in storage layer

Part of: #11250


## Why
In a product bundle, the bundled child items can either be included in the price of the bundle or be priced individually (their prices added to the bundle price). We want to be able to display the child item price differently during order creation depending on whether the child item is priced individually.

## How

This adds a `pricedIndividually` attribute to the `ProductBundleItem` storage entity, to prepare for retrieving that value from the API and using it (in PRs #11316 and #11321).

## Testing instructions

Confirm all tests pass. (We aren't handling the new attribute in other layers yet.)

You can also build and run the app as a confidence check, using the testing instructions in #11321.

---
